lib/foodcritic/api.rb

```diff
@@ -179,17 +179,21 @@ def file_match(file)
     # These are equivalent:
     #
     #     find_resources(ast)
-    #     find_resources(ast, :type => :any)
+    #     find_resources(ast, type: :any)
     #
-    # Restrict to a specific type of resource:
+    # Restrict to a specific type of resource(s):
     #
-    #     find_resources(ast, :type => :service)
+    #     find_resources(ast, type: 'service')
+    #     find_resources(ast, type: %w(service file))
     #
     def find_resources(ast, options = {})
       options = { type: :any }.merge!(options)
       return [] unless ast.respond_to?(:xpath)
       scope_type = ""
-      scope_type = "[@value='#{options[:type]}']" unless options[:type] == :any
+      unless options[:type] == :any
+        type_array =  Array(options[:type]).map! { |x| "@value='#{x}'" }
```

Comment:
You probably want `map` instead of `map!` here. If an array is passed in, this would try to actually mutate it (i.e. `Array(x)` doesn't copy if it's already an array).

Reply:
it was map! when it made sense before I switched things up. Let me fix that up